sdk/communication/Azure.Communication.Chat/src/Models/CommunicationCloudEnvironmentModel.cs

```diff
@@ -6,7 +6,7 @@
 namespace Azure.Communication
 {
     [CodeGenModel("CommunicationCloudEnvironmentModel")]
-    internal readonly partial struct CommunicationCloudEnvironmentModel
+    public readonly partial struct CommunicationCloudEnvironmentModel
```

Comment:
This type should not be public. The `Serialize` method would convert this `CommunicationCloudEnvironmentModel` to `CommunicationCloudEnvironment`

Reply:
replaced by internal